index.html

```diff
@@ -0,0 +1,317 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="css/style.css" rel="stylesheet">
+
+
+    <title>Ellis Ng'ona</title>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"></script>
+  <script>
+
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'G-WXBV13V4WN');
+</script>
+    
+</head>
+
+  <header>
+  <nav class="navbar">
+        <div class="logo">
+            <a href="#"> My Logo </a>
+        </div>
+            <ul class="nav-list">
+                <li class="list-item">
+                    <a href="#"> Portfolio </a>
+                </li>
+                <li class="list-item">
+                    <a href="#"> About </a>
+                </li>
+                <li class="list-item">
+                    <a href="#"> Contact </a>
+                </li>
+            </ul>
```

Comment:
- [ ] For the user experience (UX) purpose, replace the `href="#"`, with the id or class that contains the section you need to see, for example if the user needs to send a message, he or she can use contact menu to go there instead of scrolling down.